charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-objects-registration-config.yaml

```diff
@@ -17,6 +17,14 @@ data:
       sleep 2
     done;
 
+    function get_http_status {
+      HTTP_RESPONSE = "$1"
```

Comment:
This will result in `line 11: HTTP_RESPONSE: command not found`. It should be `HTTP_RESPONSE="$1"`.